### PR TITLE
Added useProductImage hook

### PR DESCRIPTION
**Pull request by MickhaelSh**
Added useProductImage hook , if a product doesn't have an image everywhere is showing a default placeholder. Removing from cart and wishlist deleted products

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/orders/cart/cart-item/cart-item.js [hunk 1]
```diff
@@ -45,8 +48,10 @@ const CartItem = ({ item, setModalVisibility, setModalItem, cartOperations, prom
...
     changeSizeConstructor,
     getProductPriceWithPromoCode,
-    getProductPrice
+    getProductPrice,
+    removeFromCart
   } = cartOperations;
+  const { image, checkImage } = useProductImage();
 
   const onChangeQuantity = useCallback(
     _.debounce((value) => {
```
Comment (Pipuss) [inline src/containers/orders/cart/cart-item/cart-item.js:57]: We have useDebounce hook 
You can use it if you want)

File: src/containers/orders/cart/cart-item/cart-item.js [hunk 2]
```diff
@@ -106,7 +118,7 @@ const CartItem = ({ item, setModalVisibility, setModalItem, cartOperations, prom
 
   const defaultProductImg = (
     <Link to={`${pathToProducts}/${cartItem?._id}`}>
-      <img className={styles.itemImg} src={`${IMG_URL}${itemFoto} `} alt='product-img' />
+      <img className={styles.itemImg} src={image} alt='product-img' />
```
Comment (Pipuss) [inline src/containers/orders/cart/cart-item/cart-item.js:121]: Isn't it better practice to use imageUrl naming for src attribute?

File: src/containers/orders/cart/cart-item/cart-item.js [hunk 3]
```diff
@@ -73,11 +78,18 @@ const CartItem = ({ item, setModalVisibility, setModalItem, cartOperations, prom
   };
 
   const cartItem = isFromConstructor ? constructorCartItem : product?.getProductById;
-
   const itemFoto = isFromConstructor
     ? cartItem?.model?.images?.medium
     : cartItem?.images.primary.medium;
 
+  useEffect(() => {
+    checkImage(itemFoto, isLightTheme);
```
Comment (Tolik170) [inline src/containers/orders/cart/cart-item/cart-item.js:86]: maybe rename itemFoto to itemImage?

File: src/containers/orders/cart/cart-item/cart-item.js (hunk 1 above)
Comment (MickhaelSh) [reply src/containers/orders/cart/cart-item/cart-item.js]: I haven't wanted but I used it)

File: src/containers/orders/cart/cart-item/cart-item.js (hunk 2 above)
Comment (MickhaelSh) [reply src/containers/orders/cart/cart-item/cart-item.js]: fixed

**Review by tarasSanotskyi — APPROVED** [review]

**Review by Tolik170 — DISMISSED** [review]

**Review by Pipuss — APPROVED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Pull request merged**